src/Environments/MacOS/ResourceFork.cs

```diff
@@ -342,7 +343,115 @@ public void Dump()
                 platform.A5World = a5world;
                 platform.A5Offset = jt.BelowA5Size;
                 segmentMap.AddSegment(a5world);
+
+                // Check for %A5Init CODE Segment and unpack global data to A5World
+                foreach ( var a5dataseg  in segmentMap.Segments.Values)
+                {
+                    if (a5dataseg.Name.Length >= 12 )
+                    {
+                        if (a5dataseg.Name.Substring(5,7) == "%A5Init")
+                        {
+                            var a5data = a5dataseg.MemoryArea;
+                            var a5dr = a5data.CreateBeReader(0);
+                            var a5d0 = a5dr.ReadBeUInt32();
+                            var a5d1 = a5dr.ReadBeUInt16();
+                            var a5dhdroffset = a5dr.ReadBeUInt16();
+                            if (a5d0 == 0x48e77ff8 && a5d1 == 0x49fa && a5dhdroffset < a5dataseg.MemoryArea.Length)
+                            {
+                                a5dr.Seek(a5dhdroffset - 2);
+                                var a5dbelow = a5dr.ReadBeUInt32();
+                                var a5dbanksize = a5dr.ReadBeUInt32();
+                                var a5doffset = a5dr.ReadBeUInt32();
+                                var a5dreloc = a5dr.ReadBeUInt32();
+                                var a5dhdrend = a5dr.ReadBeUInt32();
+                                if (a5dbanksize == 0x00010000)
+                                {
+                                    var a5worldbaseoffset = a5world.MemoryArea.BaseAddress + jt.BelowA5Size;
+                                    var a5beloww = new BeImageWriter(a5world.MemoryArea, jt.BelowA5Size - a5dbelow);
```

Comment:
Suggest calling this variable `a5belowWriter` to make it clearer what it is used for.